rpc/src/v1/helpers/secretstore.rs

```diff
@@ -24,6 +24,7 @@ use jsonrpc_core::Error;
 use v1::helpers::errors;
 use v1::types::EncryptedDocumentKey;
 use tiny_keccak::Keccak;
+use tiny_keccak::Hasher;
```

Comment:
would be nice to group the imports
```suggestion
use tiny_keccak::{Keccak, Hasher};
```